### PR TITLE
Communication list

### LINKED ISSUE #434: Communication list
This issue will track the creation of a communication list component. This component will be restricted to one student.

### PR DESCRIPTION
**Pull request by jitsedesmet**
This PR aims to close #434.

### CODE REVIEW

**Comment by jitsedesmet** [comment]
After all communication PR's are merged. The will be a cleanup PR.

File: frontend/locales/en/common.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ const common = {
     "additional communication info": "additional communication info",
     "communication content": "content of the communication",
     medium: "medium",
+    "by medium": "by",
```
Comment (ruvmello) [inline frontend/locales/en/common.ts:88]: is this correct?

**Review by ruvmello — CHANGES_REQUESTED** [review]

File: frontend/locales/en/common.ts (hunk 1 above)
Comment (jitsedesmet) [reply frontend/locales/en/common.ts:88]: Yes

Comment (andpuydt) [reply frontend/locales/en/common.ts:88]: The "by medium" is confusion, but it is correct in the sentences "by email", "by sms", ...

Comment (ruvmello) [reply frontend/locales/en/common.ts:88]: Okay I understand what you mean

**Review by ruvmello — APPROVED** [review]

**Pull request merged**